### PR TITLE
fix: Security fix for bouncycastle

### PR DESCRIPTION
**Pull request by adkumar1**
- To fix the below issue:
Bouncy Castle for Java before 1.73 contains a potential Denial of Service (DoS) issue within the Bouncy Castle org.bouncycastle.openssl.PEMParser class. This class parses OpenSSL PEM encoded streams containing X.509 certificates, PKCS8 encoded keys, and PKCS7 objects. Parsing a file that has crafted ASN.1 data through the PEMParser causes an OutOfMemoryError, which can enable a denial of service attack.

### CODE REVIEW

**Review by carslen — COMMENTED** [review]
Why bumping the minor version of the Helm Chart, instead of bumping the patch version to 10? To fix a security issue the application version should be bumped as well.

In addition it would be nice if you could provide clear instructions about expected build workflow runs and GH releases. Otherwise committers will have to wild guess next steps.

**Comment by adkumar1** [comment]
> Why bumping the minor version of the Helm Chart, instead of bumping the patch version to 10? To fix a security issue the application version should be bumped as well.
> 
> In addition it would be nice if you could provide clear instructions about expected build workflow runs and GH releases. Otherwise committers will have to wild guess next steps.

Hi @carslen  i thought next incremental version after 2.1.9 will be 2.2.0
Now i have updated it to 2.1.10 . Thanks

File: CHANGELOG.md
Comment (carslen) [inline CHANGELOG.md:1]: v2.1.8 doesn't exists. Please change L10 to 2.1.8 and todays date. Move L8 to 2.1.8 release. Please reconsider the entry for "Trivy security fix for bouncycastle" as your are not fixing Trivy except of Trivy workflow won't fail after PR merge. Use descriptive entries in CHANGELOG (same for the PR title btw.).

**Review by carslen — CHANGES_REQUESTED** [review]

File: CHANGELOG.md
Comment (adkumar1) [reply CHANGELOG.md:1]: Updated both

**Review by carslen — APPROVED** [review]

**Pull request merged**
